exporter/opentelemetry-exporter-otlp/src/opentelemetry/exporter/otlp/exporter.py

```diff
@@ -54,9 +54,22 @@
 ExportServiceRequestT = TypeVar("ExportServiceRequestT")
 ExportResultT = TypeVar("ExportResultT")
 
+_ENVIRON_TO_COMPRESSION = {
+    None: None,
+    "gzip": Compression.Gzip,
+    "deflate": Compression.Deflate,
+}
 
-class OTLPCompression(enum.Enum):
-    gzip = "gzip"
+
+def environ_to_compression(environ_key: str) -> Optional[Compression]:
+    environ_value = environ.get(environ_key)
```

Comment:
Might want to `.lower().strip()` it or at least `strip()?

Reply:
done